.github/workflows/check.yml

```diff
@@ -141,8 +141,6 @@ jobs:
 
   # sanitizers currently require nightly https://github.com/rust-lang/rust/issues/39699
   sanitize:
-    # disabled due to https://github.com/private-attribution/ipa/issues/837
```

Comment:
this seems to be from another PR (#842)

Reply:
I will clean this up, I think it was included through a merge